Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -168,6 +170,11 @@ private let routers: [Router<Route>] = [
     %> queryParam("payload", .appDecrypted >>> payload(.uuid >>> .tagged, ._rawRepresentable))
     <% end,
 
+  .expressUnsubscribeReply
+    <¢> post %> lit("newsletters") %> lit("express-unsubscribe-reply")
+    %> formBody(MailgunForwardPayload.self, decoder: formDecoder).map(.signatureVerification)
```

Comment:
first we decode the post body into a custom data type, and then use a `(MailgunForwardPayload) -> MailgunForwardPayload` partial iso to verify the signature that mailgun sent us.